Quests/Descriptions/quest_00160300.csv

```diff
@@ -1,14 +1,14 @@
 Title#0,"""Force Basics"""
-Explanation#0,"""■訓練メニュー<br>――――――――――――――――――――<br>課題１：テクニックのジャストアタック<br>課題２：テクニックの連続ジャストアタック<br>課題３：チャージTECのジャストアタック<br>課題４：タリスの使用<br>課題５：実戦訓練"""
+Explanation#0,"""■Training Program<br>――――――――――――――――――――<br>Task 1: Technique Just Attacks<br>Task 2: Consecutive Technique JAs<br>Task 3: Charged Technique JAs<br>Task 4: Using Talises<br>Task 5: Live combat"""
```

Comment:
JAs?

Reply:
Just Attacks doesn't quite fit and the first task was already about Just Attacks so I reckon it's fine

Reply:
ok, adding "JAs" to Misc word list